lib/modules/accountSwitcher.js

```diff
@@ -434,7 +455,38 @@ function _notifySwitchedAccountElsewhere() {
 		message,
 	});
 }
-
+async function autoSwitched() {
+	return Date.now() - await lastAutoSwitched.get() < 6000;
+}
+function autoSwitcher(thing) {
+	const currentUserIsAuthor = thing.getAuthor() === loggedInUser();
+	autoSwitched().then(isAutoSwitched => {
```

Comment:
This seems like an expensive way to check, as `lastAutoSwitched` is fetched from Storage for every post.